.github/workflows/cli_config.yml

```diff
@@ -42,3 +42,16 @@ jobs:
 
       - run: dart test
         if: ${{matrix.run-tests}}
+
+      - name: Install coverage
+        run: dart pub global activate coverage
+        if: ${{ matrix.sdk == 'stable' }}
+      - name: Collect coverage
+        run: dart pub global run coverage:test_with_coverage
+        if: ${{ matrix.sdk == 'stable' }}
+      - name: Upload coverage
+        uses: coverallsapp/github-action@50c33ad324a9902697adbf2f92c22cf5023eacf1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          path-to-lcov: ./pkgs/cli_config/coverage/lcov.info
```

Comment:
Ah, this whole workflow is running w/ a cwd of `pkgs/cli_config` (see line 23), so the path to this file should likely just be `coverage/lcov.info`.

Reply:
Apparently the path was correct already: https://github.com/dart-lang/tools/actions/runs/4505648410/jobs/7931615860?pr=44

Reply:
Hmm, that's confusing, but I agree that this PR seems to upload coverage and the other one doesn't.